job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1660,6 +1660,29 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+    /**
+     * Generates trend report for SLOCCount and cloc.
+     *
+     * @since 1.41
+     */
+    @RequiresPlugin(id = 'sloccount', minimumVersion = '1.20')
+    void sloccount(@DslContext(SlocCountContext) Closure closure) {
+        SlocCountContext context = new SlocCountContext()
+        ContextHelper.executeInContext(closure, context)
+
+        publisherNodes << new NodeBuilder().'hudson.plugins.sloccount.SloccountPublisher' {
+            if (context.pattern != '') {
+                pattern(context.pattern)
+            }
+            if (context.encoding != '') {
+                encoding(context.encoding)
+            }
+            numBuildsInGraph(context.numBuildsInGraph)
+            commentIsCode(context.commentIsCode)
```

Comment:
I wonder whether I can update the minimum Version to 1.18 and force the commentIsCode value only if the version is newer than 1.19.
```
            if (jobManagement.getPluginVersion('sloccount')?.isNewerThan(new VersionNumber('1.19'))) {
                commentIsCode(context.commentIsCode)
            }
```
Does it make sense?